Assets/Scripts/Statemachine/States/PlayerIdleState.cs

```diff
@@ -1,20 +1,34 @@
 using UnityEngine;
 
 public class PlayerIdleState : BaseState {
+
+	private bool _attack = false;
```

Comment:
Isn't it better to store this in the PlayerController?

Reply:
can be implemented later on